app/models/user.rb

```diff
@@ -69,6 +84,20 @@ def forget
     update_attribute(:remember_digest, nil)
   end
 
+  def follow(other_user)
+    following << other_user
```

Comment:
toán tử `<<` có tác dụng gì trong trường hợp này

Reply:
toán tử `<<` ở đây có tác dụng thêm 1 phần tử vào mảng vì ta đã coi `following` như 1 mảng

Reply:
sao `following` lại là 1 mảng hở em??

Reply:
vì `user.following` trả về 1 mảng của các User đang theo dõi nên ta coi đó là 1 mảng ạ

Reply:
trong trường hợp này là `self.following` em nhé
`following << other_user` sẽ làm 2 việc:
- thêm phần tử vào mảng các user đang theo dõi
- thêm bản ghi vào bảng relationships 
như này mới đầy đủ em nhé